website/docs/guides/best-practices/how-we-mesh/mesh-2-structures.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Deciding how to structure your mesh"
+description: Getting started with dbt Mesh patterns
+hoverSnippet: Learn how to get started with dbt Mesh
+---
+
+## Exploring mesh patterns
+
+Building a dbt Mesh is not a one-size-fits-all process. In fact, it's the opposite, it's about customizing your project structure to fit _your_ team and _your_ data. Often we've had to fit the data team and project structure into our company's org chart, or manage everything in one project to handle the constraints of our data and warehouse. dbt Mesh allows us to mold our organizational knowledge graph to our organizational people graph, bringing people and data closer together rather than compromising one for the other. Let's explore some language for discussing the design of these patterns.
```

Comment:
> Often we've had to fit the data team and project structure into our company's org chart, or manage everything in one project to handle the constraints of our data and warehouse

this sentence hit my ear a little weird -- seems like the main idea is how we structured dbt project(s) but the first clause seems to talk about shaping teams.